packages/swapper/src/swappers/PortalsSwapper/getPortalsTradeQuote/getPortalsTradeQuote.ts

```diff
@@ -129,10 +129,13 @@ export async function getPortalsTradeQuote(
         target: allowanceContract,
         feeAmount,
         gasLimit,
+        feeToken,
       },
       tx,
     } = portalsTradeOrderResponse
 
+    const feeAsset = feeToken === inputToken ? sellAsset : buyAsset
```

Comment:
[suggestion]: We normally use `feeAsset` for the network fee asset, so to avoid confusion maybe we can try:
```suggestion
    const protocolFeeAsset = feeToken === inputToken ? sellAsset : buyAsset
```

Reply:
Good call! Fixed in https://github.com/shapeshift/web/pull/8280/commits/3325af900eedd6cc049a2a25b8e5e6a415563384